src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenUpdateCaseStateAat.java

```diff
@@ -54,9 +56,11 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             .build();
     }
 
-    public boolean isEnvironmentAat() {
+    public boolean isEnvironmentAatOrPreview() {
         String environment = System.getenv().getOrDefault("ENVIRONMENT", null);
-        return null != environment && environment.equalsIgnoreCase(ENVIRONMENT_AAT);
+        String clusterName =  System.getenv().getOrDefault("CLUSTER_NAME", null);
+        return null != environment && (environment.equalsIgnoreCase(ENVIRONMENT_AAT) || environment.equalsIgnoreCase(ENVIRONMENT_PREVIEW))
+            || clusterName != null && (clusterName.contains(ENVIRONMENT_AAT) || clusterName.contains(ENVIRONMENT_PREVIEW));
     }
```

Comment:
wouldn't be easier to check if environment is not PROD ?

Reply:
yeah, if that's fine to allow its use on demo etc..